File: go/1080.go
package main

import "fmt"

func main() {
  var n, pos, maior int
  for i := 1; i <= 100; i++ {
    fmt.Scanf("%d", &n)
    if n > maior {
      maior = n
      pos = i
    }
  }

  fmt.Printf("%d\n%d\n", maior, pos)
}

File: lua/1080.lua
local n, pos, maior = 0, 0, 0

for i = 1, 100 do
    n = tonumber(io.read("*n"))
    if n > maior then
        maior = n
        pos = i
    end
end

print(string.format("%d\n%d", maior, pos))
